src/mon.c

```diff
@@ -1156,9 +1156,18 @@ meatbox(struct monst *mon, struct obj *otmp)
     }
 }
 
-#define mstoning(obj)                                       \
-    (ofood(obj) && (touch_petrifies(&mons[(obj)->corpsenm]) \
-                    || (obj)->corpsenm == PM_MEDUSA))
+static boolean mstoning(const struct obj *obj)
```

Comment:
As this is static, what is the benefit of having early prototype? Regarless added and noted that quite amount of prototypes are missing here in general.

Reply:
I don't know.  It may not be necessary, I have just noted it seems to be usually done and figured it may be some kind of compatibility thing (and/or a holdover from ages long past, maybe).  Maybe a DevTeam member might know if it's actually something that is held to be generally required (stylistically or otherwise).